networking/openstack/ipv6-dual-stack.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Configure OpenStack for dual stack or IPv6 only
```

Comment:
This page is mostly wrong, in so far as it appears to talk about IPAM.  IPAM in OpenStack is not handled by Calico, so none of our IPAM config is relevant.

Only the material after the "Howto" heading is relevant, and the title should be "Prepare a VM guest OS for IPv6"

Reply:
Fixed.